base-theme/assets/css/variables.scss

```diff
@@ -17,6 +17,7 @@ $medium-light-gray: #e9e9e9;
 $medium-gray: #9c9d9e;
 $dark-gray: #3d3d3d;
 $partial-collapse-height: 6em;
+$normal-text-font-size: 18px;
```

Comment:
Can we change this to be `1rem;`?

Reply:
Sure. 
1rem=16px, right? 
Though 16px is the default size, not sure but I doubt it might look a bit smaller in our course content.
@pdpinch What do you suggest?

Reply:
`1rem` equals whatever the `font-size` property is on the `body` element.  So, if that's `16px`, then it's `16px`

Reply:
Okay, if the font-size is equal to that of `body` element then do we need to separately give the same font-size to these elements?
I think, what we can do is to decide a `font-size` for the `body` element, (either 16px or 18px) and remove these explicit font-sizes
![image](https://user-images.githubusercontent.com/93309234/155673687-240ddd0a-8eeb-4ed5-810c-b46db58ca4c3.png)
So that all of these elements and other elements can be of the same font-size (i.e: font-size of body).
For `h1`, `h2`, `h3` and few other elements, we can replace their font-sizes with `rem`. 

What do you think?

Reply:
yes please. simpler is better.

Reply:
Okay, what should be the font-size of `body` then, 16px or 18px? Currently it's `16px`.
Note: This font-size will also be applied to `p`, `li`, `table` and other elements unless explicitly defined.